[ET-901] G20.09.1 Hotfix for AR Modal problems (#1773)
moderntribe/event-tickets: readme.txt

== Changelog ==

= [5.0.1] 2020-08-31 =

> * Fix - Prevent the Attendee Registration modal from incorrectly setting RSVP as the provider class. [TBD]

Author: ```suggestion
* Fix - Prevent the Attendee Registration modal from incorrectly setting RSVP as the provider class. [ET-901]
```

> * Fix - Prevent the RSVP provider class from interfering with the Attendee Registration modal flow in regards to a ticket's real provider. [TBD]

Reviewer: Doesn't feel concise. Maybe an improvement:

```suggestion
* Fix - Prevent the Attendee Registration modal from incorrectly setting RSVP as the provider class. [TBD]
```
